src/app/screens/BookDetail/components/BookDetail/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { View, Image, Text, TouchableOpacity } from 'react-native';
+import styles from './styles';
+import  defaultImage from '../../../../assets/img_book6.png';
+
+
+function BookDetail({title, author, imageSource, year, type}) {
+    return (
+        <View style={styles.container}>
+            <View style={styles.bookContainer}>
+                <Image style={styles.image} resizeMode={'cover'} source={imageSource||defaultImage}></Image>
+                <View style={styles.infoContainer}>
+                    <Text style={styles.title} numberOfLines={1}>{title}</Text>
+                    <View style={styles.dataContainer}>
+                        <Text style={styles.text}>{author}</Text>
+                        <Text style={styles.text}>{year}</Text>
+                        <Text style={styles.text}>{type}</Text>
```

Comment:
since you are repeating the same declaration 3 times you can create an array with this values and then mapping them inside here like:
```js
const aux = [author, year, type];
return( ...
{aux.map(value => <Text style={styles.text}>{value}</Text>)}
...
```